backend/main.py

```diff
@@ -1,23 +1,33 @@
 from fastapi import FastAPI
 
-from databases.postgres import PostgresConnector
 from utils import logger
 from routers import base
-from routers.v1 import chat, embedding_model
+from routers.v1 import embedding_model
 from settings import Constants
 
-logger = logger.LoggerFactory(__name__).get_logger()
+logger = logger.LoggerFactory().get_logger(__name__)
 
-app = FastAPI(
-    title=Constants.API_NAME,
-    version=Constants.API_VERSION,
-    description=Constants.API_DESCRIPTION,
-)
 
-postgres_instance = PostgresConnector.get_instance()
+def create_app() -> FastAPI:
+    """
+    Construct and configure the FastAPI application
+    """
+    # Initialize FastAPI application
+    app = FastAPI(
+        title=Constants.API_NAME,
+        version=Constants.API_VERSION,
+        description=Constants.API_DESCRIPTION,
```

Comment:
Same here, should be FastApiConfig

Reply:
Check out my reply [here](https://github.com/TinhHoaSolutions-EzHR/chatbot/pull/11#discussion_r1833114010)